modules/api/api-xsjs/pom.xml

```diff
@@ -20,6 +20,11 @@
 			<artifactId>dirigible-commons-api</artifactId>
 			<version>${dirigible.version}</version>
 		</dependency>
+    <dependency>
+      <groupId>com.sap.cloud.sdk.cloudplatform</groupId>
+      <artifactId>scp-cf</artifactId>
+      <version>3.60.0</version>
```

Comment:
Please extract the version as a property in the main **pom.xml** and use it here as **${scp-cf.version}**